Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -138,7 +355,20 @@ The Content Security Policy for a particular site can be disabled with the
 Site-specific Content-Security-Policy endpoints
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The disposition-specific property `reportingUrl` can either be:
+The reporting endpoint is used to receive browser reports about violations to
+the security policy, for example if a YouTube-URL was requested, but could
+not be displayed in an iframe due to a directive not allowing this.
+
+Reports like this can help to gain insight, what URLs are used by editors
+and might need inclusion into the policy.
+
+Since reports can be sent by any browser, they can possibly easily flood
+a site with requests and take up storage space. Reports are stored in the
+:sql:`sys_http_report` database table.
+
+To influence whether this endpoint accepts reports,
+the disposition-specific property `reportingUrl` can be configured and
+set to either:
```

Comment:
available since v12.4.27/v13.4.5: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.4.x/Important-105856-AllowSite-specificContent-Security-PolicyEndpoints.html

This should be added, as this is fairly new.

Reply:
Added a version note, thanks!